src/verifiers/documentStoreIssued/openAttestationEthereumDocumentStoreIssued.test.ts

```diff
@@ -171,15 +325,24 @@ describe("openAttestationEthereumDocumentStoreIssued", () => {
         name: "OpenAttestationEthereumDocumentStoreIssued",
         type: "DOCUMENT_STATUS",
         data: {
-          details: [
-            {
-              address: "0x8Fc57204c35fb9317D91285eF52D6b892EC08cD3",
-              issued: false
+          details: {
+            address: "0x8Fc57204c35fb9317D91285eF52D6b892EC08cD3",
+            issued: false,
+            reason: {
+              code: 1,
+              codeString: "DOCUMENT_NOT_ISSUED",
+              message:
+                "Certificate 0x76cb959f49db0cffc05107af4a3ecef14092fd445d9acb0c2e7e27908d262142 has not been issued under contract 0x8Fc57204c35fb9317D91285eF52D6b892EC08cD3"
             }
-          ],
+          },
           issuedOnAll: false
         },
-        message: "Certificate has not been issued",
+        reason: {
```

Comment:
leaving a comment here because i said its duplicative and you explained why but then told me to leave a comment anyway idkwhy

Reply:
will leave :D